backend/app/crud/event.py

```diff
@@ -33,6 +33,8 @@ def get_list(start: date | None, end: date | None) -> list[models.Event]:
     if end is not None:
         query = query.where(models.Event.date <= end)
 
+    query = query.order_by(models.Event.date.desc())
```

Comment:
I switched this around. Having the newest events at the top seemed like a sensible default :)